microprofile-rest-client/README.adoc

```diff
@@ -50,6 +50,7 @@ include::../shared-doc/run-arquillian-tests.adoc[leveloffset=+1]
 
 // Run the Quickstart in Red Hat CodeReady Studio or Eclipse
 include::../shared-doc/run-the-quickstart-in-jboss-developer-studio.adoc[leveloffset=+1]
+include::../shared-doc/build-and-run-the-quickstart-with-openshift.adoc[leveloffset=+1]
```

Comment:
- please move the bootable jar section include to be before the openshift one
- please remove the hacks specific to this QS in the included bootable jar section, see
-- https://github.com/wildfly/quickstart/blob/main/shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc?plain=1#L47
-- https://github.com/wildfly/quickstart/blob/main/shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc?plain=1#L61
-- https://github.com/wildfly/quickstart/blob/main/shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc?plain=1#L73